app/controllers/users/edit.rb

```diff
@@ -0,0 +1,27 @@
+# Show the form to edit a user's profile
+get '/users/:id/edit' do
+  @user = User.find(params[:id])
+  erb :'users/edit'
+end
+
+# Route to update a user's profile
+put 'users/:id' do
+  @user = User.find(params[:id])
+  @user.assign_attributes(params[:user])
+
+  if @user.save
+    # If saved succesfully, go back to user's profile page
+    redirect "/users/#{@user.id}"
+  else
+    erb :'/users/edit'
+  end
+end
+
```

Comment:
All routes for users (controller) should be placed within a single rb file called `users.rb`